src/Pikaday.js

```diff
@@ -40,6 +40,17 @@ var ReactPikaday = React.createClass({
     }
   },
 
+  // user props to pass down to the underlying DOM node
+  getRestProps: function() {
```

Comment:
@ximus Maybe we can be a bit more specific and call it `getDomProps`, since all that should be left are DOM-support props? I think that explains the intent a little better.